smoke-tests/framework/src/main/java/com/microsoft/applicationinsights/smoketest/SmokeTestExtension.java

```diff
@@ -364,8 +364,6 @@ protected Set<Integer> getLivenessCheckPorts() {
                 "APPLICATIONINSIGHTS_CONNECTION_STRING",
                 "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;"
                     + "IngestionEndpoint=http://host.testcontainers.internal:6060/")
-            .withEnv("APPLICATIONINSIGHTS_ROLE_NAME", "testrolename")
-            .withEnv("APPLICATIONINSIGHTS_ROLE_INSTANCE", "testroleinstance")
```

Comment:
I think better for defaults (except for APPLICATIONINSIGHTS_CONNECTION_STRING) to be provided by json, otherwise individual smoke tests cannot override them in their json since env vars always win